build_system/tests/test_array/good/output/aggregation_2.catala_en.scalc

```diff
@@ -0,0 +1,207 @@
+let A (A_in : A_in{"x_in": unit → array S{"id": integer; "income": money}})
+  =
+  decl x : unit → array S{"id": integer; "income": money};
```

Comment:
So about the whole test suite of clerk I'm a little skeptical ? It seems to me that here's you're testing more the Catala compiler than the build system. More precisely, these build systems tests are going to break if we change things in the Catala compiler and this would seem strange. Shouldn't the build system test suite be about comparing the expected output in terms of Ninja files? For that, isn't the simplest option to just implement in clerk a way to test clerk by invoking itself?

Reply:
I agree with you about the irrelevance of the current tests. 

> Shouldn't the build system test suite be about comparing the expected output in terms of Ninja files? For that, isn't the simplest option to just implement in clerk a way to test clerk by invoking itself?

In my opinion, the simplest way is to compare `Ninja_utils.ninja` values produce by the driver with the expected ones with frameworks like `ounit` or `alcotest`. Therefore, it will not be necessary to keep the content of the files stored in `build_system/tests/output`.

Reply:
I agree, let's use `ounit` or `alcotest`.